docs/command_docs_template.md

```diff
@@ -0,0 +1,126 @@
+---
+title: SET
+description: The `SET` command in DiceDB is used to set the value of a key. If the key already holds a value, it is overwritten, regardless of its type. This is one of the most fundamental operations in DiceDB as it allows for both creating and updating key-value pairs.
+---
+<!-- description in 2 to 3 sentences, following is an example -->
+The `SET` command in DiceDB is used to set the value of a key. If the key already holds a value, it is overwritten, regardless of its type. This is one of the most fundamental operations in DiceDB as it allows for both creating and updating key-value pairs.
+
+## Syntax
+
+```bash
+SET key value [EX seconds | PX milliseconds | EXAT unix-time-seconds | PXAT unix-time-milliseconds | KEEPTTL] [NX | XX]
+```
+
+## Parameters
+<!-- add all parameters, small description, type and required -->
```

Comment:
| Parameter   | Description                                                              | Type     | Required |
|-------------|--------------------------------------------------------------------------|----------|----------|
| `key`       | The name of the key to be set.                                            | String   | Yes      |
| `value`     | The value to assign to the key.                                           | String   | Yes      |
| `EX`        | Set the key’s expiration time, in seconds.                                | Integer  | No       |
| `PX`        | Set the key’s expiration time, in milliseconds.                           | Integer  | No       |
| `EXAT`      | Set the expiration as an absolute UNIX timestamp in seconds.              | Integer  | No       |
| `PXAT`      | Set the expiration as an absolute UNIX timestamp in milliseconds.         | Integer  | No       |
| `KEEPTTL`   | Retain the key's existing TTL when updating its value.                    | Flag     | No       |
| `NX`        | Set the key only if it does not already exist.                            | Flag     | No       |
| `XX`        | Set the key only if it already exists.

Just a thought, shall we consider adding all supported parameters info in this sample doc to give idea to contributors to consider this as well?

Reply:
Yes, we can do that. I will add these along with others. Using comments to explain and guide what to modify in each section.